src/code-templates/services/order-service/Dockerfile

```diff
@@ -1,6 +1,26 @@
-FROM node:14.2.0-alpine3.11
+FROM node:16 AS dependencies
```

Comment:
Just curious, why prefer the full-size image over stretch/alpine?

Reply:
Don't use stretch
Don't use alpine

Reply:
Why not alpine? what's the recommanded tag? slim?

Reply:
I explained some of that in short here: https://twitter.com/liran_tal/status/1564623675923566598?s=20&t=TjHrrQ3ARhCwa43PUUiugw and working on an article to document more elaborately. The other recommended tag is indeed slim.

Reply:
@lirantal my 2 cents: for 99% of the cases the `musl` is just fine, using it a few years in production and until now didn't need to recompile anything. In addition, the Alpine image is "automatically" compliant with various standards like FedRAMP which I find very convenient - no need to harden the image.

Reply:
I don't know what "99%" means because your use-cases are not everybody else's. There are issues attributed to compiling native modules (node-gyp), issues related to Python and musl, etc. Alpine image is also not officially supported by the Node.js Docker team, so they end up building it but if something breaks there's no inherent or explicit urgency to solve it.

I never heard of a "FedRAMP-approved" container images, it's more about the software within, to which I assume you are also compiling the fips openssl library?

This begs the question though - why are you using Alpine? is this cult cargo or something else?

Reply:
> I never heard of a "FedRAMP-approved" container images

ofc, but AFAIK, no need to harden the base image, which makes it easier to start with and the same as the commercial stack.

> to which I assume you are also compiling the fips openssl library

I'm not aware of it, if you could drop a link or two about this matter it would be great! 🙏 

> This begs the question though - why are you using Alpine? is this cult cargo or something else?

In my company, we used to have heavy hardened centos images so I got used to using Alpine instead. I definitely need to re-evaluate the situation now.
In addition, Alpine images are more secure because of `musl` and its small surface attack, but you're the security expert between us :) so definitely waiting for the article :)